client/src/LayoutComponents/Cohorts/Cohorts.css

```diff
@@ -0,0 +1,23 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100&display=swap");
+
+.title {
```

Comment:
@bayan-404 It might be a good idea to make this selector more specific, by adding a className to the cohorts div below and then using it here (for example `.cohorts .title {`).
`.title` is a pretty common className, and this css would be applied to other title divs in other parts of the app (it is not local to only the Cohorts component).